src/ProjectService.Data/UserRepository.cs

```diff
@@ -17,6 +19,18 @@ public UserRepository(IDataProvider provider)
             _provider = provider;
         }
 
+        public IEnumerable<DbProjectUser> GetProjectUsers(Guid projectId, bool showNotActive)
+        {
+            var predicate = PredicateBuilder.New<DbProjectUser>(u => u.ProjectId == projectId && u.IsActive);
+
+            if (showNotActive)
+            {
+                predicate.Or(u => !u.IsActive);
```

Comment:
Are you sure about this predicate? it looks like u.ProjectId == projectId && u.IsActive || !u.IsActive, so if user is inactive and his id not equal to porjectId we took it too